modules/rostests/apitests/shell32/ShellState.cpp

```diff
@@ -11,6 +11,7 @@
 #include <stdio.h>
 #include <shellutils.h>
 #include <strsafe.h>
+#include <Shlwapi.h>
```

Comment:
Should be lowercase `s` for linux machines

Reply:
fixed